python/packages/core/agent_framework/ollama/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import importlib
+from typing import Any
+
+PACKAGE_NAME = "agent_framework_ollama"
+PACKAGE_EXTRA = "ollama"
+_IMPORTS = [
+    "OllamaChatClient",
+    "OllamaSettings",
+    "__version__",
+]
+
+
+def __getattr__(name: str) -> Any:
+    if name in _IMPORTS:
+        try:
+            return getattr(importlib.import_module(PACKAGE_NAME), name)
+        except ModuleNotFoundError as exc:
+            raise ModuleNotFoundError(
+                f"The '{PACKAGE_EXTRA}' extra is not installed, "
+                f"please do `pip install agent-framework[{PACKAGE_EXTRA}]`"
```

Comment:
we moved away from extras, mention `pip install agent-framework-ollama` instead.